server/BraveAlerterConfigurator.js

```diff
@@ -130,6 +135,24 @@ class BraveAlerterConfigurator {
     return new Location(installations[0].name, SYSTEM.BUTTONS)
   }
 
+  createHistoricAlertFromRow(row) {
+    const alertType = row.num_presses > 1 ? ALERT_TYPE.BUTTONS_URGENT : ALERT_TYPE.BUTTONS_NOT_URGENT
+    return new HistoricAlert(row.id, row.unit, row.incident_type, alertType, row.num_presses, row.created_at, row.responded_at)
+  }
+
+  // Historic Alerts are those with status "Completed" or that were last updated longer ago than the SESSION_RESET_TIMEOUT
```

Comment:
This comment, or something like it, would be helpful in `db.js` as well to explain what the complicated Postgres query is doing.

Reply:
Done!

Reply:
thanks!